fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -118,9 +118,11 @@ class FastlaneCrash < StandardError
     # raised from user_error!
     class FastlaneError < StandardError
       attr_reader :show_github_issues
+      attr_reader :error_info
```

Comment:
I'd keep the `error_info` property as is, for consistency, and because you're showing an underlying error.